src/components/Form/ProfileForm.tsx

```diff
@@ -27,17 +27,17 @@ function ProfileForm() {
   const { open: openConnectModal } = useWeb3Modal();
   const { user } = useContext(TalentLayerContext);
   const provider = useProvider({ chainId: import.meta.env.VITE_NETWORK_ID });
-  const userDetails = useUserDetails(user?.uri);
+  const userDescription = user?.id ? useUserById(user?.id)?.description : null;
   const { data: signer } = useSigner({ chainId: import.meta.env.VITE_NETWORK_ID });
 
-  if (user?.uri && !userDetails) {
+  if (!user?.id) {
     return <Loading />;
   }
 
   const initialValues: IFormValues = {
-    title: userDetails?.title || '',
-    about: userDetails?.about || '',
-    skills: userDetails?.skills || '',
+    title: userDescription?.title || '',
+    about: userDescription?.about || '',
```

Comment:
we need to remove everything that is not linked to this project, it's not really a fork, it's just a copy of the stack & structure from indie

Reply:
globally we never need to sync both